custom_components/kia_uvo/translations/en.json

```diff
@@ -26,6 +26,119 @@
           "unknown":"Unexpected error"
        }
     },
+    "services": {
+      "force_update": {
+        "name": "Force Update",
+        "description": "Force your vehicle to update its data. All vehicles on the same account as the vehicle selected will be updated.",
+        "fields": {
+          "device_id": {
+            "name": "Speed",
```

Comment:
@cdnninja  Is "Speed" the correct value?
The other "device_id name" fields have "Vehicle" as the value.

Reply:
Corrected.  Thanks for catching that.